srcpkgs/NetworkManager/template

```diff
@@ -35,7 +35,7 @@ license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/NetworkManager"
 changelog="https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/raw/nm-1-48/NEWS"
```

Comment:
```suggestion
changelog="https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/raw/nm-1-50/NEWS"
```

Reply:
Thank you for pointing that out.

Fixed. :+1: